Formula/yubico-piv-tool.rb

```diff
@@ -13,14 +13,21 @@ class YubicoPivTool < Formula
   end
 
   depends_on "check" => :build
+  depends_on "cmake" => :build
+  depends_on "gengetopt" => :build
+  depends_on "help2man" => :build
+  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on "check"
   depends_on "openssl@1.1"
+  depends_on "pcsc-lite"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    ENV.append_to_cflags "-I#{Formula["pcsc-lite"].opt_include}/PCSC"
```

Comment:
Is there no way to tell cmake this directly?

Reply:
We could pass `-DCMAKE_C_FLAGS=-I#{Formula["pcsc-lite"].opt_include}/PCSC` to cmake

Reply:
Sounds better to me.